src/mame/dataeast/dec8.h

```diff
@@ -17,216 +17,299 @@
 #include "screen.h"
 #include "tilemap.h"
 
-class dec8_state : public driver_device
+class base_state : public driver_device
```

Comment:
The name `base_state` is too generic for a class in the global namespace.  You can only use generic names like that in anonymous namespaces.  Please use `dec8_state_base` instead.